api/src/Entity/Repository/UserRepository.php

```diff
@@ -206,20 +206,36 @@ public function findInactive($select = null)
     }
 
     /**
-     * @return User[]
+     * @return array
      */
     public function findActiveLaysInLastYear()
     {
-        $oneYearAgo = (new DateTime())->modify('-1 Year');
-
-        $qb = $this->createQueryBuilder('u');
-        $qb
-            ->select()
-            ->where('u.lastLoggedIn > :login_cutoff')
-            ->andWhere('u.roleName = :lay_deputy_role')
-            ->setParameter('login_cutoff', $oneYearAgo)
-            ->setParameter('lay_deputy_role', User::ROLE_LAY_DEPUTY);
-
-        return $qb->getQuery()->getResult();
+        $oneYearAgo = (new DateTime())->modify('-1 Year')->format('Y-m-d');
+
+        $conn = $this->getEntityManager()->getConnection();
+
+        $sql = <<<SQL
+SELECT u.id,
+u.firstname as user_first_name,
+u.lastname as user_last_name,
+u.email as user_email,
+u.phone_main as user_phone_number,
+u.registration_date,
+u.last_logged_in,
+c.firstname as client_first_name,
+c.lastname as client_last_name,
+COUNT(r.id) as submitted_reports
+FROM dd_user as u
+LEFT JOIN deputy_case as dc on u.id = dc.user_id
+LEFT JOIN client as c on dc.client_id = c.id
+LEFT JOIN report as r on c.id = r.client_id
+WHERE r.submit_date is not null AND u.role_name = 'ROLE_LAY_DEPUTY' AND u.last_logged_in > :oneYearAgo
```

Comment:
Just a random query, but what fidelity do we need this data in? Do we want > :oneYearAgo or >= :oneYearAgo

Reply:
Good point - its anyone whos logged in within the last year. I'll whack an `=` on.